test/unit/all_content_workflow_presenter_test.rb

```diff
@@ -1,29 +1,69 @@
 require "test_helper"
 
 class AllContentWorkflowPresenterTest < ActiveSupport::TestCase
-  should "provide a CSV export of content workflow" do
+  setup do
+    @transaction_created_at_time = Time.zone.now - 2.days
     transaction_user = FactoryBot.create(:user, :govuk_editor, name: "Ray Khan")
+    transaction_edition = FactoryBot.create(:transaction_edition, title: "Register to vote (armed forces)",
+                                                                  slug: "register-to-vote-armed-forces", created_at: @transaction_created_at_time,
+                                                                  state: "published", assigned_to_id: transaction_user.id)
```

Comment:
Could we maybe reformat this (and the `guide_edition` line) to avoid having to scroll sideways (and only have a single attribute per line)?
```suggestion
    transaction_edition = FactoryBot.create(
      :transaction_edition,
      title: "Register to vote (armed forces)",
      slug: "register-to-vote-armed-forces",
      created_at: @transaction_created_at_time,
      state: "published",
      assigned_to_id: transaction_user.id)
```